### PR TITLE
Pages

### PR DESCRIPTION
**Pull request by vaishnavipaliya11**
The layout of the page is given which contains scroll bar , navigation and display cards. 
For now dummy cards are added which will be refactored later.
image

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,10 +1,27 @@
 import "./App.css";
+import Mockman from "mockman-js";
+import { Playlist } from "./Pages/Playlist/Playlist";
+import { WatchLater } from "./Pages/Watch/Watch";
+import { Liked } from "./Pages/Liked/Liked";
+import { History } from "./Pages/History/History";
+import {Explore} from "./Pages/Explore/Explore";
```
Comment (znjs) [inline src/App.js:7]: can make a index file in pages and import everything in one line

File: src/Components/Navbar.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
+.nav-container{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-content: center;
+    align-items: center;
+    /* box-shadow: rgba(0, 0, 0, 0.24) 0px 3px 8px; */
```
Comment (znjs) [inline src/Components/Navbar.css]: can remove unwanted code

File: src/Components/Navbar.jsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  return (
+    <div className="nav-container">
+      <header className="nav-heading">
+        <nav>
+          <h2>
+            {" "}
+            <FaInfinity />
+            Infinity video
+          </h2>
+        </nav>
```
Comment (znjs) [inline src/Components/Navbar.jsx:20]: can wrap the whole component in `nav` since all the tags are part of nav-bar

**Review by znjs — APPROVED** [review]
- Good folder structure
- Liked that you have implemented media-queries

**Review by ragini07 — APPROVED** [review]
-good folder structure
-Suggestion - can use barrel export
-rest is good to go

**Pull request merged**